src/components/NextButton.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <button @click="onClick" :class="getButtonStyle()"><slot></slot></button>
```

Comment:
```suggestion
  <button @click="onClick" :class="getButtonStyle()"></button>
```

불필요한 slot 이 있는 것 같네요. 제거 부탁드립니다.

Reply:
React의 children 개념으로 사용해서 버튼의 내용을 전달해보려고 했습니다 :)~!